frontend/src/locales/de.json

```diff
@@ -191,7 +191,9 @@
     },
     "couldNotForge": "Schmieden nicht möglich: Zu wenig Guthaben oder Transaktion wurde abgelehnt.",
     "couldNotReforge": "Umschmieden nicht möglich: Zu wenig Guthaben, Staub oder Transaktion wurde abgelehnt.",
-    "couldNotBurn": "Verbrennen nicht möglich: Zu wenig Guthaben oder Transaktion wurde abgelehnt."
+    "couldNotBurn": "Verbrennen nicht möglich: Zu wenig Guthaben oder Transaktion wurde abgelehnt.",
+    "disableDynamicMintingSalvage": "An dieser Kette kann jeweils eine Waffe geborgen werden",
+    "disableDynamicMintingForge": "An dieser Kette kann nur ein Schmiedevorgang gleichzeitig durchgeführt werden"
   },
```

Comment:
can you change the translation for "disableDynamicMintingSalvage"
to:
"In diesem Netzwerk kannst du deine Waffen nur einzeln einschmelzen!"
and
"disableDynamicMintingForge" to
"In diesem Netzwerk kannst du Waffen nur einzeln schmieden!"

Reply:
delete all other translations. We use english as a fallback. You don't have to copy all english strings to the files :-)